exist-core/src/test/java/org/exist/xquery/functions/transform/ImportStylesheetTest.java

```diff
@@ -57,51 +61,72 @@ private static Path getConfigFile() {
     }
 
     private static String getQuery(final String importLocation) {
-        final String xslt = "<xsl:stylesheet xmlns:xsl=\"http://www.w3.org/1999/XSL/Transform\"\n" +
+        final String xslt =
+                "<xsl:stylesheet xmlns:xsl=\"http://www.w3.org/1999/XSL/Transform\"\n" +
                 "    xmlns:xs=\"http://www.w3.org/2001/XMLSchema\"\n" +
                 "    xmlns:functx=\"http://www.functx.com\"\n" +
-                "    exclude-result-prefixes=\"xs\"\n" +
+                "    exclude-result-prefixes=\"functx xs\"\n" +
                 "    version=\"2.0\">\n" +
-                "    \n" +
                 "    <xsl:import href=\"" + importLocation + "\"/>\n" +
-                "    \n" +
                 "    <xsl:template match=\"/\">\n" +
-                "      <r>" +
+                "      <out>\n" +
                 "        <xsl:value-of select=\"functx:replace-first(., 'bonjourno', 'hello')\"/>\n" +
-                "      </r>" +
+                "      </out>\n" +
                 "    </xsl:template>\n" +
-                "    \n" +
                 "</xsl:stylesheet>";
 
-        return "transform:transform(" + inputXml + ", " + xslt + ", ())";
+        return "transform:transform(" + INPUT_XML + ", " + xslt + ", ())";
     }
 
     private static void assertTransformationResult(ResourceSet result) throws XMLDBException {
         assertNotNull(result);
         assertEquals(1, result.getSize());
-        assertEquals(expectedOutput, result.getResource(0).getContent());
+        assertEquals(EXPECTED_OUTPUT, result.getResource(0).getContent());
     }
 
     @ClassRule
     public static ExistXmldbEmbeddedServer existXmldbEmbeddedServer = new ExistXmldbEmbeddedServer(true, false, true, getConfigFile());
 
     @Test
-    public void testImportNoScheme() throws XMLDBException {
-        final String xquery = getQuery(moduleLocation);
+    public void fromRegisteredImportUri() throws XMLDBException {
+        final String xquery = getQuery("http://www.functx.com/functx.xsl");
```

Comment:
@line-o I found  a working document here: https://www.datypic.com/xsl/functx-1.0.1-doc.xsl that we may should use instead...

Reply:
I just coincidentally was _also_ available at that address and not just registered with that _namespace URI_ in exist-db's package repository.
It is non-obvious to use HTTP-addresses to uniquely name packages without _also_ guaranteeing the source is available there but this is the status-quo, not the use-case this test is about and also a different discussion.

Reply:
see also what @adamretter wrote:
> This test was to ensure the loading of XSLT imports from within EXPath packages.